R/process_data.R

```diff
@@ -62,6 +68,59 @@ process_pacta_results <- function(data, start_year, end_year, time_horizon,
   return(data_processed)
 }
 
+#' Remove rows from PACTA results that belong to company-technology combinations
+#' for which there is no information on the exposure in the portfolio. We join
+#' company results on the portfolio exposure based on the last available year of
+#' the production forecast. Hence we filter for missings in that year.
+#'
+#' @inheritParams calculate_annual_profits
+#' @inheritParams report_company_drops
+#' @param data tibble containing filtered PACTA results
+#'
+#' @return A tibble of data without rows with no exposure info
+#' @noRd
+remove_companies_with_missing_exposures <- function(data,
+                                                    start_year,
+                                                    time_horizon,
+                                                    log_path) {
+  n_companies_pre <- length(unique(data$company_name))
+
+  companies_missing_exposure_value <- data %>%
+    dplyr::filter(.data$year == .env$start_year + .env$time_horizon) %>%
```

Comment:
still i thinkt it would be a good idea to add a comment why you check for plan carsen on the last year

Reply:
added